ios/versioned-react-native/ABI33_0_0/ABI33_0_0ExpoKit.podspec

```diff
@@ -0,0 +1,98 @@
+    ss.dependency         "ABI33_0_0UMReactNativeAdapter"
+    ss.dependency         "ABI33_0_0UMBarCodeScannerInterface"
+    ss.dependency         "ABI33_0_0UMCameraInterface"
+    ss.dependency         "ABI33_0_0UMConstantsInterface"
+    ss.dependency         "ABI33_0_0UMFaceDetectorInterface"
+    ss.dependency         "ABI33_0_0UMFileSystemInterface"
+    ss.dependency         "ABI33_0_0UMFontInterface"
+    ss.dependency         "ABI33_0_0UMImageLoaderInterface"
+    ss.dependency         "ABI33_0_0UMPermissionsInterface"
+    ss.dependency         "ABI33_0_0UMSensorsInterface"
+    ss.dependency         "ABI33_0_0UMTaskManagerInterface"
+    ss.dependency         "ABI33_0_0EXAdsAdMob"
+    ss.dependency         "ABI33_0_0EXAdsFacebook"
+    ss.dependency         "ABI33_0_0EXAmplitude"
+    ss.dependency         "ABI33_0_0EXSegment"
+    ss.dependency         "ABI33_0_0EXAppAuth"
+    ss.dependency         "ABI33_0_0EXAppLoaderProvider"
+    ss.dependency         "ABI33_0_0EXAV"
+    ss.dependency         "ABI33_0_0EXBackgroundFetch"
+    ss.dependency         "ABI33_0_0EXBarCodeScanner"
+    ss.dependency         "ABI33_0_0EXBlur"
+    ss.dependency         "ABI33_0_0EXBrightness"
+    ss.dependency         "ABI33_0_0EXCalendar"
+    ss.dependency         "ABI33_0_0EXCamera"
+    ss.dependency         "ABI33_0_0EXConstants"
+    ss.dependency         "ABI33_0_0EXContacts"
+    ss.dependency         "ABI33_0_0EXCrypto"
+    ss.dependency         "ABI33_0_0EXDocumentPicker"
+    ss.dependency         "ABI33_0_0EXFacebook"
+    ss.dependency         "ABI33_0_0EXFaceDetector"
+    ss.dependency         "ABI33_0_0EXFileSystem"
+    ss.dependency         "ABI33_0_0EXFont"
+    ss.dependency         "ABI33_0_0EXGL"
+    ss.dependency         "EXGL-CPP"
+    ss.dependency         "ABI33_0_0EXGoogleSignIn"
+    ss.dependency         "ABI33_0_0EXHaptics"
+    ss.dependency         "ABI33_0_0EXImageManipulator"
+    ss.dependency         "ABI33_0_0EXImagePicker"
+    ss.dependency         "ABI33_0_0EXKeepAwake"
+    ss.dependency         "ABI33_0_0EXLinearGradient"
+    ss.dependency         "ABI33_0_0EXLocalAuthentication"
+    ss.dependency         "ABI33_0_0EXLocalization"
+    ss.dependency         "ABI33_0_0EXLocation"
+    ss.dependency         "ABI33_0_0EXMailComposer"
+    ss.dependency         "ABI33_0_0EXMediaLibrary"
+    ss.dependency         "ABI33_0_0EXPermissions"
+    ss.dependency         "ABI33_0_0EXPrint"
+    ss.dependency         "ABI33_0_0EXRandom"
+    ss.dependency         "ABI33_0_0EXSecureStore"
+    ss.dependency         "ABI33_0_0EXSensors"
+    ss.dependency         "ABI33_0_0EXSharing"
+    ss.dependency         "ABI33_0_0EXSMS"
+    ss.dependency         "ABI33_0_0EXSpeech"
+    ss.dependency         "ABI33_0_0EXSQLite"
+    ss.dependency         "ABI33_0_0EXTaskManager"
+    ss.dependency         "ABI33_0_0EXVideoThumbnails"
+    ss.dependency         "ABI33_0_0EXWebBrowser"
```

Comment:
Oh, imho we should remove these dependencies from here 🤔 ExpoKit doesn't require unimodules to work and due to these deps it's probably not possible to exclude these modules from ExpoKit projects, right? If so, let's create an issue so we don't forget.

Reply:
It’s a versioned podspec, not the published one. Revisiting `ExpoKit.podspec` dependencies is another thing to do. 💪

Reply:
Unversioned, published `ExpoKit.podspec` doesn't contain dependencies on specific modules, so no task to do!